src/server/pps/cmds/cmds_test.go

```diff
@@ -232,59 +230,6 @@ func TestJSONStringifiedNumbers(t *testing.T) {
 	require.NoError(t, tu.BashCmd(`pachctl list pipeline`).Run())
 }
 
-// TestJSONMultiplePipelinesError tests that when creating multiple pipelines
-// (which only the encoding/json parser can parse) you get an error indicating
-// the problem in the JSON, rather than an error complaining about multiple
-// documents.
-func TestJSONMultiplePipelinesError(t *testing.T) {
```

Comment:
This behavior is changing. If you send these pipeline specs to Pachyderm, you should now get an error complaining about multiple documents.